### PR TITLE
Node colors themes

### PR DESCRIPTION
**Pull request by Tara-Lakshmipathy**
Refactored so that node colors are defined in a separate file: `themes.py`. Will expand on this file later to create proper themes. I still need to figure out how to set colors in the javascript side of things from here.

### CODE REVIEW

File: pyironflow/themes.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+from pyiron_workflow.nodes.function import Function
+from pyiron_workflow.nodes.macro import Macro
+from pyiron_workflow.nodes.transform import DataclassNode
+
```
Comment (liamhuber) [inline pyironflow/themes.py:4]:
```suggestion

if typing.TYPE_CHECKING:
    from pyiron_workflow.node import Node

```

Nit: for hinting

File: pyironflow/themes.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+def node_color(node, theme):
+
+    if theme == 'light':
+        return light_mode(node)
```
Comment (liamhuber) [inline pyironflow/themes.py:9]:
```suggestion
        return light_mode(node)
    else:
        raise ValueError(f'Theme must be one of ("light",) but got {theme}')
```

This is actually not a nit from my end -- the `if` branch needs to fail cleanly here

**Review by liamhuber — CHANGES_REQUESTED** [review]
One necessary change to fail cleanly on the `if` clause, but otherwise some nits

File: pyironflow/themes.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
+from pyiron_workflow.nodes.function import Function
+from pyiron_workflow.nodes.macro import Macro
+from pyiron_workflow.nodes.transform import DataclassNode
+
+
+def node_color(node, theme):
```
[the remark this answers is not in the document]
Comment (liamhuber) [reply pyironflow/themes.py:6]: And a rename. Feel free to disregard

[the remark this answers is not in the document]
Comment (Tara-Lakshmipathy) [reply pyironflow/themes.py:8]: I'm ok with the renaming :D

**Review by liamhuber — APPROVED** [review]
Nice, lgtm.

I guess light could be the default for theme too.

**Comment by Tara-Lakshmipathy** [comment]
@jan-janssen, on this one I will leave the final merging to you. I'm not sure how this pull request plays in with the conda package (sorry, I'm a newbie for conda-forge related stuff)

**Comment by jan-janssen** [comment]
> @jan-janssen, on this one I will leave the final merging to you. I'm not sure how this pull request plays in with the conda package (sorry, I'm a newbie for conda-forge related stuff)

You can merge it. A new conda package is only created once a new release is created. The important part is that the conda package of `pyironflow` should not contain any dependency on the development version of `pyiron_workflow` but rather only depend on features released in the conda version of `pyiron_workflow`.

**Pull request merged**